<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralGroupchat</class>
 <widget class="QWidget" name="GeneralGroupchat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeneralGroupchatUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="1" column="0">
      <widget class="QGroupBox" name="alertGroupBox">
       <property name="toolTip">
        <string>Highlight lines and alert the user if this text is included in chat</string>
       </property>
       <property name="title">
        <string>Word highlighting</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>11</number>
        </property>
        <property name="topMargin">
         <number>11</number>
        </property>
        <property name="rightMargin">
         <number>11</number>
        </property>
        <property name="bottomMargin">
         <number>11</number>
        </property>
        <item>
         <widget class="QListWidget" name="lw_highlightWords">
          <property name="toolTip">
           <string>Words which will cause a notification in group chat.</string>
          </property>
          <property name="whatsThis">
           <string>This is the list of words which when said in group chat will cause the user to be notified.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_newHighlightWord">
          <property name="toolTip">
           <string>Type a new word here and press &quot;Add Word&quot; for it to be added to the highlight list</string>
          </property>
          <property name="whatsThis">
           <string>Type a new word here and press &quot;Add Word&quot; for it to be added to the highlight list</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_addHighlightWord">
            <property name="toolTip">
             <string>Adds the new word to the list</string>
            </property>
            <property name="whatsThis">
             <string>Adds the new word to the list</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_removeHighlightWord">
            <property name="toolTip">
             <string>Removes the selected word from the highlight list</string>
            </property>
            <property name="whatsThis">
             <string>Removes the selected word from the highlight list</string>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_coloringType">
       <property name="toolTip">
        <string>Select nick coloring type for conference log</string>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Automatic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Manual palette</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="ncGroupBox">
       <property name="title">
        <string>Nick coloring</string>
       </property>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>11</number>
        </property>
        <property name="topMargin">
         <number>11</number>
        </property>
        <property name="rightMargin">
         <number>11</number>
        </property>
        <property name="bottomMargin">
         <number>11</number>
        </property>
        <item>
         <widget class="QListWidget" name="lw_nickColors">
          <property name="toolTip">
           <string>Colors used for nicks in groupchat</string>
          </property>
          <property name="whatsThis">
           <string>This is the list of colours with which nicks will be drawn in group chat.</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="le_newNickColor">
            <property name="toolTip">
             <string>Type a colourcode here, or press the palette to the right</string>
            </property>
            <property name="whatsThis">
             <string>Type a colourcode here, or press the palette to the right</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_nickColor">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_addNickColor">
            <property name="toolTip">
             <string>Adds the color to the list</string>
            </property>
            <property name="whatsThis">
             <string>Adds the new color to the list</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_removeNickColor">
            <property name="toolTip">
             <string>Removes the selected color from the list</string>
            </property>
            <property name="whatsThis">
             <string>Removes the selected color from the list</string>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nick Coloring Type</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="topMargin">
      <number>20</number>
     </property>
     <item row="1" column="0">
      <widget class="QCheckBox" name="ck_showJoins">
       <property name="text">
        <string>Show joins</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="ck_showStatusChanges">
       <property name="text">
        <string>Show status changes</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="ck_showStatusPriority">
       <property name="text">
        <string>Show status with priority</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="ck_showInitialJoins">
       <property name="text">
        <string>Show initial joins</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>le_newHighlightWord</tabstop>
  <tabstop>pb_removeHighlightWord</tabstop>
  <tabstop>pb_addHighlightWord</tabstop>
  <tabstop>le_newNickColor</tabstop>
  <tabstop>pb_nickColor</tabstop>
  <tabstop>pb_removeNickColor</tabstop>
  <tabstop>pb_addNickColor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
